prime-router/docs/api/token.yml

```diff
@@ -9,29 +9,29 @@ info:
 paths:
   /api/token:
     post:
-      description: generate a new token for the requested scope
+      description: Generate and return a new token for the requested scope
       security:
         - OAuth2: [ user ]
-      parameters:
```

Comment:
The openapi spec has been updated to reflect that parameters should be sent in the body.